build_torch_xla_libs.sh

```diff
@@ -43,8 +43,8 @@ else
   cp -r -u -p $THIRD_PARTY_DIR/xla_client $THIRD_PARTY_DIR/tensorflow/tensorflow/compiler/xla/
 
   pushd $THIRD_PARTY_DIR/tensorflow
-  bazel build $MAX_JOBS $VERBOSE --define framework_shared_object=false -c "$MODE" "${OPTS[@]}" $XLA_CUDA_CFG \
-    //tensorflow/compiler/xla/xla_client:libxla_computation_client.so
+  bazel build $MAX_JOBS $VERBOSE --define framework_shared_object=false -c "$MODE" --copt="-D_GLIBCXX_USE_CXX11_ABI=0" \
```

Comment:
Check the diff I sent, WRT this coming from torch._C -> setup.py ...

Reply:
Done, thanks!